docs/user/discover.asciidoc

```diff
@@ -3,114 +3,227 @@
 
 [partintro]
 --
+**_Tell {kib} where to find your data, then search and filter it for hidden insights and relationships._**
 
-When you know what your data includes, you can create visualizations
-that best display that data and build better dashboards.
-*Discover* enables you to explore your data, find
-hidden insights and relationships, and get answers to your questions.
+You’ve added your data, and now you’re ready to dig in. You have questions about your data.
+What events were logged most recently? What pages on your website contain a
+specific word or phrase? What processes on your server take longer than 500 milliseconds to respond?
+This tutorial shows you how to use {kib} *Discover* to quickly search large amounts of
+data and understand what’s going on at any given time.
 
-With *Discover*, you can:
+You’ll learn to:
 
-* Access every document in every index that matches your selected index pattern
-* Search your data and filter the search results
-* Get field-level details about the documents that match your search
-* View the events that occurred just before and after a document
+- **Select** data for your exploration, including setting a time range,
+searching with the {kib} query language, and limiting results with filtering options.
+- **Explore** the details of your data, view individual documents, and create tables
+that summarize the contents of the data.
+- **Present** your findings in reports and visualizations.
+
+At the end of this tutorial, you’ll be ready to start exploring with your own
+data in *Discover*.
 
 [role="screenshot"]
 image::images/Discover-Start.png[Discover]
 
 
 [float]
-[[select-pattern]]
-=== Set up your index pattern
+=== Prerequisites
+
+- If you don’t already have {kib}, set it up with https://www.elastic.co/cloud/elasticsearch-service/signup?baymax=docs-body&elektra=docs[our free trial].
+- You must have data in {es}.  This guide uses the
+<<gs-get-data-into-kibana,ecommerce sample data set>>, but you can use your own data.
+- You should have an understanding of {ref}/documents-indices.html[{es} documents and indices].
 
-The first thing to do in *Discover* is to select an <<index-patterns, index pattern>>, which
-defines the data you want to explore and visualize.
-If you haven't yet created an index pattern, you can add a <<add-sample-data, sample data set>>,
-which has a pre-built index pattern.
 
 [float]
-=== Set a time filter
+[[whats-you-goal-in-discover]]
+=== What’s your goal?
 
-By default, *Discover* shows data for the last 15 minutes.
-If you have a time-based index, and no data displays,
-you might need to increase the time range. Using the <<set-time-filter, time filter>>,
-you can specify a common or recently-used time range, a relative time
-from now, or an absolute time range.
+Most users have one of two goals in mind when exploring their data:
 
-[float]
-=== Search your data
+- **Get an overview of what is happening.**
```

Comment:
It would be cool if these goals were tied to the tutorial. Like a nice set up for what questions they'll have answered when they get to the bottom of the page.